01-ruby-fundamentals/exercises/admissions-program.md

```diff
@@ -0,0 +1,45 @@
+# Admissions Program
+
+![waves](../images/waves.jpg)
+
+This exercise will be good for you if you want more practice with:
+- creating logical boolean statements
+- practicing Ruby conditional syntax
+- constructing code from a word problem
+- practice debugging complex scenarios
```

Comment:
These bullets should probably not start with "practice", since that's how the sentence above leads into them.